src/DataWrangling/restoring.jl

```diff
@@ -283,30 +281,30 @@ Keyword Arguments
 
 - `rate`: The restoring rate, i.e., the inverse of the restoring timescale (in s⁻¹).
 
-- `dir`: The directory where the native ECCO data is located. If the data does not exist it will
-         be automatically downloaded. Default: `download_ECCO_cache`.
+- `dir`: The directory where the native obs/reanalysis data is located. If the data does not exist it will
+         be automatically downloaded. No default, must be provided.
```

Comment:
```suggestion
- `dir` (required): The directory where the native data is located. If the data does not exist it will
         be automatically downloaded.
```